tests/integration/insights/test_importer.py

```diff
@@ -105,6 +114,71 @@ def test_get_nutrition_table_predictions(self):
         }
 
 
+class TestImageOrientationImporter:
```

Comment:
Those are good tests but those are unit tests not integration tests, because here you don't use the database at all.

The integration test should create prediction in the db and call the general import insight method and verify insights are created as expected.

So you should moved these tests to complete your unit tests (because you test more cases here) and possibly add a real integration test.